tests/suite/custom_resources_utils.py

```diff
@@ -249,6 +279,117 @@ def create_virtual_server_from_yaml(
         raise
 
 
+def create_ts(custom_objects: CustomObjectsApi, yaml_manifest, namespace) -> dict:
```

Comment:
Minor: can we change it to `create_ts_from_yaml` just to keep it inline with other lib functions.

Reply:
done